pypeit/telescopes.py

```diff
@@ -44,6 +44,17 @@ def __init__(self):
                                                diameter=10,
                                                eff_aperture=72.3674)
 
+class SubaruTelescopePar(TelescopePar):
+    def __init__(self):
+        loc = EarthLocation.of_site('Subaru Telescope')
+        super(SubaruTelescopePar, self).__init__(name='SUBARU',
+                                               longitude=loc.lon.to(units.deg).value,
+                                               latitude=loc.lat.to(units.deg).value,
+                                               elevation=loc.height.to(units.m).value,
+                                               fratio=12.2,  # CASSEGRAIN
+                                               diameter=8.2,
+                                               eff_aperture=72.3674) # TODO -- FIX THIS??
```

Comment:
is the `eff_aperture` ok?  if so, remove the TODO
  if not, fix it  :)

Reply:
I need to find out whether this value is reasonably correct. Asking around...